roles/ceilometer-common/templates/etc/serverspec/ceilometer_spec.rb

```diff
@@ -8,11 +8,13 @@
   end
 end
 
-files = ['api_paste.ini', 'event_pipeline.yaml', 'policy.json',
-  'ceilometer.conf', 'event_definitions.yaml', 'pipeline.yaml']
-files.each do |file|
+files = {'api_paste.ini'=> 640, 'event_pipeline.yaml'=> 640, 'policy.json'=> 644,
```

Comment:
do  we really care about the mode of any of these files?  I get ceilometer because it may contain passwords, but the others?